README.md

```diff
@@ -108,9 +107,54 @@ print(response.text)
 # Interpret your results using the API - where is the space station right now ?
 ```
 
+### Finding the number of people in space
+
+Open Notify has one more API endpoint, `/astros.json`. It tells you how many people are currently in space. The format of the responses can be studied [HERE](http://open-notify.org/Open-Notify-API/People-In-Space/).
```

Comment:
Instead of "one more" here (since this is no longer the last example), let's use the old language from the second example. In other words, 
> Let's repeat the above for the second endpoint, `astros.json`.

Here I also removed the leading `/` for the sake of consistency.